### ISSUE TITLE
some infos in the animation pannel 

### ISSUE THREAD

**Issue by mizka76** [tannerhelland/PhotoDemon#326]
hello 

I choose to make a strict translation for french now .  Before, as PD is a freeware , easy to install , with useful info boxes , ( main reason of my interest as teacher for young people for it ) and as people often begin on PCs to try a software to see if they can use it easily, I try sometimes to translate so that the item was easily understood adding parenthesis, I removed some of them  - ( I remenber my beginnings , alone with my machine,  on windows 98 where french people have to clic on "start" to close the pc (!!) ) 

... some words about translating: "copy merged" is in french "copier avec fusion" (cf Photoshop menus, this is the strict translation in french , but it does not tell the noobs that it is the visible layers thet will get merged, ) "copier visible" (french translation in Gimp) or "copier les couches visibles" (artrage)   did not says that it will be merged in the clipboard  .   So I choose a mix " copier avec fusion les calques visibles"  " copy merged  of the visible layers " 

- there is no more an icon in the menu for "paste to new image" ...?
- is it possible to put  in the "animation options" pannel that appears after clicking on the menu image> "animation" some indications : 
nota:  
- two image layers or more are needed to do an animation
- named layers will be renamed with (interval) added
- for saving animation use EXPORT in the menu "files"   > this one I think  should be usefull , beginners often don't know all the differences between image file formats and animation formats . 

the french update :
[French.zip](https://github.com/tannerhelland/PhotoDemon/files/5192285/French.zip)
Updating the stable version 8 will be marvellous for me, my pupils , I will be able to teach them the basics of animation ( as i did in my young time with paintshop pro7 and animation shop !)  many thanks!

**Comment by tannerhelland** [comment]
Hello @mizka76 .  I apologize that it has taken me a few days to reply.  I wanted to give this issue proper consideration before responding.

And before I forget, thank you very much for updated French text!  _Merci beaucoup!_

> there is no more an icon in the menu for "paste to new image" ...?

I am currently working on modernizing some of PhotoDemon's icons.  It is not always possible to find existing free icons that meet the requirements of a project like PD, so sometimes I have to create my own, and they do not always turn out very well.  

This is one of many menus where I simply haven't found an icon that I like yet, nor have I taken the time to make a replacement.  I hope to ship a number of new and improved icons in the next release, so hopefully an icon will be ready by then!

> is it possible to put in the "animation options" pannel that appears after clicking on the menu image> "animation" some indications :

I like your ideas for clarifying behavior on the Animation Options panel, but I have concerns about placing the text directly on the panel.

Localizing PhotoDemon is a complex task.  Because translations across different languages vary so much in length, it is almost impossible to design a layout that works across e.g. Chinese, English, French, and German.  For this reason, I have actively reduced the text content of many panels over the years, to try and minimize layout issues.  

In my opinion, the ideal place for additional explanation on a dialog like this is a proper Help file.  I have been working on a Help system for PhotoDemon for some years, but it is not ready yet.  (It is a huge undertaking, and documenting PhotoDemon's hundreds of different panels is no easy task.)

My plan is to put a "help" button directly on the command bar at the bottom of each dialog, which links to an online page with screenshots and detailed explanations of all panel behaviors.  This will finally give me a way to explain many nuances of PhotoDemon's various tools, including in-depth technical discussion of many features, without worrying about the layout issues specific to application panels.

I say all this because at present, I prefer to place your suggested text inside such a Help page, rather than on the panel itself.  Hopefully that is okay.

> for saving animation use EXPORT in the menu "files" > this one I think should be usefull , beginners often don't know all the differences between image file formats and animation formats .

In current nightly builds this disclaimer is no longer necessary, because PhotoDemon will automatically switch between e.g. Export Animated GIF and Save Normal GIF panels depending on the state of the current image.  I will leave the Export > Animated... menus behind as a convenience (and a way for people to discover that PhotoDemon supports animated formats), but use of that menu is no longer *required* for saving animated files - just click "Save", and PhotoDemon will sort out the rest!

> Updating the stable version 8 will be marvellous for me, my pupils , I will be able to teach them the basics of animation ( as i did in my young time with paintshop pro7 and animation shop !) many thanks!

You are most welcome.  I am very excited that PhotoDemon is useful in a classroom setting, especially in this strange year of COVID-19.  🙏

**Comment by mizka76** [comment]
Hello ! 
thanks a lot for your explanations !  as I just read your answer for the batch problem with Webp , I understand that  my demand of informing to export as gif an animation was unuseful as PdD chooses the good format automatically . 

thank you too to explain what you planned , I understand that it is a lot of time to write help files, and , if I can say, it is not urgent as many and enough informations are in the menus and pannels . Creating icons is also long to do !
About this :  I think that Photodemon 8 is a marvellous tool to begin in photo editing ( I hoped such a software for years : no installation, strong possibilities, and  instructive info pannels and menus that prevent clicking everywhere to test what happens : anybody can use the program, in autonomy , without tutorial)
 so , this suggestion :  why not keeping this perfect Photodemon 8 as a "basic edition" for the beginners on your site , and developping this 9th version aside as a future  "pro" version  ?  am i wrong ?  ( when I see all the work you do on PD , all these new options or tools that  you create, I think this should be your intention) 

- for the translation files : you are too fast !!  you always add new things or updates ! It is hard to follow !  :-) 
thank you very much to have updated the french langage file , it is much better , because when you simplified ( and compiled i supposed) , some very minor oddities of the google translation reappears, or maybe some lines that I missed  ( "in" for inches ( pouces) for example were translated in french as "dans" ( into) .  So I begin to read again line by line the traduction file, of the updated 8 version to be sure that the syntax is good . I translated the new phrases of the 9 version , I will send this week end .

- a very little thing : you simplified with only one "none" ( for 'animation repétitions' and 'text font antialiased') in english or Us it is a neutral word, but in latin languages , words are feminine or masculine and adjective are accorded to them : 
animation (feminine word) > none  = should be translated "aucune" in french, but, for 
antialiasing   ( masculine ) the french translation should be " aucun"
there is only for this term "none" that the syntax in french  is not correct in one case . ( but every body understands....)

I do like your new menu in the new interface , "showing recent files" is a so useful  and good idea .
the verbose image interface in the 8 ("basic édition" !) is useful because it mentions that "drag and drop" is possible. 
the fact that this info dissapears in  the new and more pro interface is not a problem, a pro user knows this.

Thanks for all !!!
